modules/OCPP/OCPP.cpp

```diff
@@ -372,12 +371,9 @@ void OCPP::init() {
                 this->charge_point->stop_session(connector, ocpp1_6::DateTime(timestamp), energy_Wh_import);
                 this->charge_point->plug_disconnected(connector);
             } else if (event == "Error") {
-                auto error = session_events["error"];
-                if (error == "OverCurrent") {
-                    this->charge_point->error(connector, ocpp1_6::ChargePointErrorCode::OverCurrentFailure);
-                } else {
-                    this->charge_point->vendor_error(connector, error);
-                }
+                auto evse_error = session_events["error"];
```

Comment:
in general, always use const for constant variables.  With json dicts, please use .at(), because the bracket notation might just create an empty dict item, which is not what we want and also hard to debug. So the line should look like
const auto evse_error = session_events.at("error")

Reply:
I will make a separate PR for the syntactical and framework stuff, since that affects many places in the file

Reply:
ok